README.md

```diff
@@ -39,45 +37,23 @@ You don’t have to ever use `eject`. The curated feature set is suitable for sm
 
 # Firebase setup
 
+For this project, you need an existing firebase project and a domain with gSuite for business.
```

Comment:
nit: the proper spelling of gSuite is G Suite. Also, I don't think "for business" is needed. The G Suite subscription that we have is G Suite Basic.

```suggestion
For this project, you need an existing Firebase project and a domain with G Suite.
```